tools/metams/metaMS_plot/metaMS_plot.xml

```diff
@@ -0,0 +1,242 @@
+        <param name="selectedtic" type="boolean" checked="false" falsevalue="false" truevalue="true" label="Do you want to process for TIC(s) ?" />
+        <param name="selectedbpc" type="boolean" checked="false" falsevalue="false" truevalue="true" label="Do you want to process for BPC(s) ?" />
+
+        <conditional name="selectedeic">
+            <param name="EIC" type="select" label="Do you want to process for EIC(s) ?">
+                <option value="false">NO</option>
+                <option value="true">YES</option>
+            </param>
+            <when value="true">
+                <param name="unkn" type="text" value="1:5" label="EIC_Unknown" help="vector of peaks number to be plotted, for example 1:5 (mean 1 to 5) or 1,4,12  (means 1 4 and 12). For all EIC use 0" /> 
+            </when>
+            <when value="false"/>
+        </conditional>
+
+        <expand macro="input_file_load"/>
+
+    </inputs>
+
+    <outputs>
+    
+        <!-- Only when user selected TIC option -->
+        <data name="ticsRawPdf"   format="pdf" from_work_dir="TICs_raw.pdf" label="TICs_raw.pdf">
+            <filter>selectedtic is True</filter>
+        </data>
+        <!-- Only when user selected BPC option -->
+        <data name="bpcsRawPdf"   format="pdf" from_work_dir="BPCs_raw.pdf" label="BPCs_raw.pdf">
+            <filter>selectedbpc is True</filter>
+        </data>
+        <!-- Only when user selected EIC option -->
+        <data name="unknownPdf"   format="pdf" from_work_dir="GCMS_EIC.pdf" label="GCMS_EIC.pdf">
+            <filter>selectedeic['EIC'] is True</filter>
+        </data>
+
+    </outputs>
+
+    <tests>
+
+        <!-- Test TIC -->
+        <test expect_num_outputs="1">
+            <param name="metaMS" value="runGC_test.RData"  ftype="rdata" />
+            <expand macro="test_file_load_alg_symlink" />
+            <param name="selectedtic" value="true"/>
+            <output name="ticsRawPdf" value="TICs_raw.pdf" ftype="pdf" compare="sim_size" delta="600"/>
+        </test>
+        <!-- Test BPC -->
+        <test expect_num_outputs="1">
+            <param name="metaMS" value="runGC_noDB.RData"  ftype="rdata" />
+            <expand macro="test_file_load_alg_symlink" />
+            <param name="selectedbpc" value="true"/>
+            <output name="bpcsRawPdf" value="BPCs_raw.pdf" ftype="pdf" compare="sim_size" delta="2000"/>
+        </test>
+        <!-- Test EICs -->
+        <test expect_num_outputs="5">
+            <param name="metaMS" value="runGC_noDB.RData"  ftype="rdata" />
+            <expand macro="test_file_load_alg_symlink" />
+            <conditional name="selectedeic">
+                <param name="EIC" value="true"/>
+                <param name="unkn" value="1:5"/>
+            </conditional>
+            <output name="unknownPdf" value="GCMS_EIC_noDB.pdf" ftype="pdf" compare="sim_size" delta="1200"/>
```

Comment:
Instead of `sim_size` you could use a `<has_size size="xyz" delta="1200"/>` content assertion. Then we do not need to save the actual file in the repo.